test/test_models/create_dummy_model.py

```diff
@@ -0,0 +1,128 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+# Licensed under the MIT License.  See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+Create dummy ONNX models that contain only inputs and outputs
```

Comment:
Could you also add a comment here detailing why a dummy model with input output names might be needed?

Reply:
Added